draft-ietf-gnap-core-protocol.md

```diff
@@ -4833,6 +4833,8 @@ not its value can be sent. This allows the AS to use pre-registered symmetric ke
 as key derivation schemes to take advantage of symmetric cryptography but without requiring
 key distribution at runtime, which would expose the keys in transit.
 
+The security of GNAP when in use with symmetric keys relies on the cryptographic security of the keys themselves. Therefore, a key management system MUST be used to supply the keys. Moreover, the symmetric key MUST NOT be a human memorable password or a value derived from one.
```

Comment:
Security considerations shouldn't have normative requirements. This text is good, but it needs to be up in the main body. Maybe in the section on key proofing and key material?

Reply:
Target a new section 7.1.2 for this text.